packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md

```diff
@@ -14,57 +14,64 @@ When it comes to product experiences, no one should get left behind. That's why
+
+Users with low vision can have different needs depending on the nature of their visual impairment. Users may have difficulty with color differentiation, blurriness, or lack of vision in central or peripheral areas.
+
+- **Tip:** These needs mean that interfaces should not rely on color alone to communicate information, palettes need to have sufficient contrast, and layouts should be responsive when font sizes are increased.
 <br />
 
-### Cognitive 
-Users who have difficulty processing information benefit from well-written content. 
-- Information should be clear, concise, and easy to scan. 
+### Cognitive
+
+Users who have difficulty processing information benefit from well-written content.
+
+- Information should be clear, concise, and easy to scan.
 - Consider visual hierarchy, chunk content into short, related sections, and avoid long paragraphs.
 <br />
 <br />
 
-These are some of the main user groups, but it is not an exhaustive list. Often “the Persona Spectrum”, coined by Microsoft, can be used to understand related mismatches and motivations across a spectrum of permanent, temporary, and situational scenarios. 
+These are some of the main user groups, but it is not an exhaustive list. Often “the Persona Spectrum”, coined by Microsoft, can be used to understand related mismatches and motivations across a spectrum of permanent, temporary, and situational scenarios.
 
-While accessibility tends to focus on those with disabilities, everyone benefits from accessible products. For example: 
+While accessibility tends to focus on those with disabilities, everyone benefits from accessible products. For example:
 
-- Accessible written content is easier to comprehend for people who aren’t fluent in the language and for people with low literacy levels. 
-- Alternative text (alt text) makes images accessible to users with low bandwidth connections or with older technologies that can’t load the images. 
-- Closed captioning benefits those in crowded areas, those teaching children how to read, or those learning a new language. 
+- Accessible written content is easier to comprehend for people who aren’t fluent in the language and for people with low literacy levels.
+- Alternative text (alt text) makes images accessible to users with low bandwidth connections or with older technologies that can’t load the images.
+- Closed captioning benefits those in crowded areas, those teaching children how to read, or those learning a new language.
 
 There are plenty more examples we could include. Essentially, accessibility takes _all users_ into account.
 
-<img src="./PF-a11y.png" alt="Accessibility spectrum example" />
+The image below shows that accessibility is a spectrum. Different kinds of cognitive and physical impairments impact how users interact with products. It is important to take all these differing disabilities into account when building accessible products.
```

Comment:
A `pf-v5-u-screen-reader` class on this paragraph might make more sense. I don't think it needs to be a visible paragraph. WDYT, @jessiehuff ?

Reply:
Great idea @nicolethoen - added `pf-v5-u-screen-reader`

Reply:
I'm ok with either approach! When I added that paragraph, I think I was just trying to give context to that image in general, but I'm ok with making it only visible to screen readers if it doesn't make sense to include for visual users/is already obvious.